### PR TITLE
meew0 mode ratelimit

### PR DESCRIPTION
**Pull request by deansheather**
Simplifies #2, because @george99g's implementation was over-complicated. Idea from @george99g.

### CODE REVIEW

**Comment by deansheather** [comment]
@SteamingMutt please respond.

**Comment by Dougley** [comment]
Since when am I assigned lmao, but I'll check it out

**Comment by Dougley** [comment]
@deansheather Looks good to me 👌

File: assets/space.js [hunk 1]
```diff
@@ -166,7 +166,17 @@ var celestial = function () {
 
 // Eyes
 var eyesCount = 0;
+var meewBlocked = false;
+var meewBlockAsked = false;
 var doMeew = function () {
+  if (meewBlocked) return;
+  if (eyesCount > 4999 && !meewBlockAsked) {
+    meewBlockAsked = true;
+    meewBlocked = confirm('meew0 mode has a ratelimit set, because b1nzy. ¯\_(ツ)_/¯ Would you like to disable this ratelimit?');
```
Comment (Zeludon) [inline assets/space.js:175]: `¯\\_(ツ)_/¯` Plz

Unless you enjoy

Comment (Dougley) [inline assets/space.js:175]: 👀 👌

**Pull request merged**